### DISCUSSION TITLE
dhcpcd 9.x and Networkmanager

### DISCUSSION THREAD

**Discussion by maravtdm** [rsmarples/dhcpcd#56] [General]
A recent NetworkManager update suggests using the dhcpcd plugin with version >= 9.3.3 and the --noconfigure option.
Can anyone give me advice on where to place this option so that it is used by NM?

NM version 1.32.10
dhcpcd version : 9.4.0
And I have : /etc/NetworkManager/conf.d/00-dhcp-client.conf
with dhcp=dhcpcd

**Comment by rsmarples** [comment]
Sorry for the late reply.
NM will pass --noconfigure to dhcpcd, and that option was introduced in dhcpcd-9.3.3

So you don't need to do anything.
